.eslintrc.json

```diff
@@ -58,7 +58,10 @@
         "@typescript-eslint/comma-dangle": ["error", "only-multiline"],
         "@typescript-eslint/explicit-module-boundary-types": "off",
         "@typescript-eslint/ban-ts-comment": "off",
-        "@typescript-eslint/ban-types": "off"
+        "@typescript-eslint/ban-types": "off",
+        "react/require-default-props": "off",
+        "react/no-unused-prop-types": "off",
+        "react/jsx-no-bind": "off"
```

Comment:
`react/jsx-no-bind` is a useful rule, it can prevent some nasty hard to debug errors with losing `this`.

Since `eslint` is only unhappy about it in 2 places, it's better to fix it.

* `LocationSelector/LocationSelector.tsx`
* `LocationSelector/LocationSelectorDialog/LocationSelectorDialog.tsx`

```ts
// Rewrite handler functions from functions to arrow functions.
// Example:
// src/components/LocationSelector/LocationSelector.tsx

// Before
function handleCloseModal() {
    if (isOpen && isBrowserTabVisible) {
        setIsOpen(false);
        onBlur?.();
    }
}

// After
const handleCloseModal = () => {
    if (isOpen && isBrowserTabVisible) {
        setIsOpen(false);
        onBlur?.();
    }
};
```

Reply:
:+1: